app/views/RoomView/UploadProgress.tsx

```diff
@@ -210,7 +212,7 @@ class UploadProgress extends Component<IUploadProgressProps, IUploadProgressStat
 				key={item.path}
 				style={[
 					styles.item,
-					index !== 0 ? { marginTop: 10 } : {},
+					index !== 0 ? { marginTop: 2 } : {},
```

Comment:
Can you explain why you're changing the visual of a logic bug?
And why you're not using multiples of 4?